README.md

```diff
@@ -1,13 +1,107 @@
 # SAnsible
 
-Ansible roles
+This repo is a set of tools to aid with the development of roles in the SAnsible organisation.
 
 
 
 
-## Maintenance
+## The roles
 
-To release new version of any of the SAnsible role please run `./bin/release.sh`
-from inside the role's folder.
+All roles in SAnsible follow a similiar pattern, however some roles are slightly older than
+others so their Makefiles, tests and travis.yml files are somewhat out of date.
 
-Run `./bin/report.sh` to check the status of all sansible roles.
+In general though the roles follow a standardised pattern.
+
+* Tests are conducted using Vagrant locally and travis for PRs and branches
+* Proper YAML syntax should be used everywhere
+* Each role has an editorconfig file to ensure proper formating
+* The Makefile contains useful targets for running the tests
+* Three tags are present in each role: build, configure and assert
```

Comment:
I would say
 "Two tags are present in each role: `build` and `configure`. The test.yml is using additional tag `assert`."